include/clap/ext/draft/preset-load.h

```diff
@@ -16,7 +16,10 @@ typedef struct clap_plugin_preset_load {
    // The preset discovery provider defines the uri and load_key to be passed to this function.
```

Comment:
"uri" should now be "location"

Reply:
Good catch!
I also fixed a few other place where we used URI instead